.travis.yml

```diff
@@ -18,13 +18,13 @@ script:
   - bundle exec rubocop
   - bundle exec brakeman
 deploy:
-  provider: elasticbeanstalk
-  access_key_id: $AWS_ACCESS_KEY_ID
-  secret_access_key: $AWS_SECRET_ACCESS_KEY
-  region: us-east-1
-  app: move.mil
-  env: staging
-  bucket_name: elasticbeanstalk-us-east-1-321940898149
+  - provider: elasticbeanstalk
+    access_key_id: $AWS_ACCESS_KEY_ID
+    secret_access_key: $AWS_SECRET_ACCESS_KEY
+    region: us-east-1
+    app: move.mil
+    env: move-mil-staging
+    bucket_name: elasticbeanstalk-us-east-1-321940898149
```

Comment:
Changing to this YML syntax that would allow for us to set up Travis to do different deploys based on conditions. Check out [Travis' deployment documentation](https://docs.travis-ci.com/user/deployment/) for more on what the options are there.